sdk/storage/azure-storage-blobs/test/ut/blob_container_client_test.cpp

```diff
@@ -227,6 +227,62 @@ namespace Azure { namespace Storage { namespace Test {
     EXPECT_TRUE(std::includes(listBlobs.begin(), listBlobs.end(), p1Blobs.begin(), p1Blobs.end()));
   }
 
+  TEST_F(BlobContainerClientTest, ListBlobsFlatPageable)
+  {
+    const std::string prefix1 = RandomString();
+    const std::string baseName = "blob";
+
+    std::set<std::string> blobs;
+
+    for (int i = 0; i < 10; ++i)
+    {
+      std::string blobName = prefix1 + baseName + std::to_string(i);
+      auto blobClient = m_blobContainerClient->GetBlockBlobClient(blobName);
+      auto emptyContent = Azure::Core::IO::MemoryBodyStream(nullptr, 0);
+      blobClient.Upload(emptyContent);
```

Comment:
Wait, why would we allow a nullptr memorystream? What is the expectations when calling `blobClient.Upload` like that?

This issue would add input validation with either abort or throw if the argument was null.
https://github.com/Azure/azure-sdk-for-cpp/issues/1769